src/ingameop.cpp

```diff
@@ -611,6 +593,30 @@ static bool startIGOptionsMenu()
 	return true;
 }
 
+void intReopenMenuWithoutUnPausing(bool toOptionsMenu)
+{
+	if (toOptionsMenu)
+	{
+		startIGOptionsMenu();
+		return;
+	}
```

Comment:
I'd prefer if you didn't move this entire function down a few lines. That way the diff can be looked over faster as to what changed.

Reply:
yeah I moved the function below because I added a call to another function which was below that one so it was unreachable

Reply:
True. I didn't notice that, heh. :+1: